website/docusaurus.config.js

```diff
@@ -117,6 +117,10 @@ module.exports = {
                         url: 'https://crawlee.dev/python/api/class/RequestQueue',
                         group: 'Classes',
                     },
+                    {
+                        url: 'https://crawlee.dev/python/api/class/Request',
+                        group: 'Classes',
+                    },
```

Comment:
I already added this.

https://github.com/apify/apify-sdk-python/blob/master/website/docusaurus.config.js#L120:L124

Reply:
Cool! As I expected, it caused some rendering issues with the reexported `Request` docs (missing inherited symbols). To fix this, we need to deploy Crawlee docs with the latest version of the plugin (and then rebuild SDK docs, preferably with the changes from this PR).

![obrazek](https://github.com/user-attachments/assets/3b6f3220-070a-4d2f-880b-99fe18f29146)